back/src/utils/nodeProcessReport.ts

```diff
@@ -43,7 +42,7 @@ export const startPeriodicNodeProcessReport = (
     logger.info(
       makeReport(
         currentCpuUsage,
-        (currentTime.getTime() - previousTime.getTime()) * 10 * cpus().length,
+        (currentTime.getTime() - previousTime.getTime()) * 1000,
```

Comment:
on a pas besoin de multiplier par le nombre de CPU ?

Reply:
alors j'ai vue plusieurs exemple qui ne l'utilise pas et en comparant les log les resultats font carément plus sens comme ça mais après c'est pas super claire dans la doc de node si cpuUsage() renvois la somme totale des temps cpu

Reply:
en local, je suis pas sûr que ça ait d'incidence (cpus().length sera certainement égal à 1), mais dans les instances scalingo ou d'autres cloud, cpus().length sera peut être supérieur ?

Reply:
je laisse @bbohec et @JeromeBu nous éclairer :)

Reply:
```suggestion
        (currentTime.getTime() - previousTime.getTime()) * 1000  * cpus().length,
```

On veut la consomation cpu sur la capacité CPU total de la machine et pas juste sur le thread où tourne le processus node.

Reply:
en fait le fait de multiplier par mille plutôt que par cent dans le cas des valeur de ce matin par exemple ça nous donnerai un usage cpu a 0.1 % ce qui est beaucoup plus bas que ce qu'on avait sur graphana. Et sur la doc de node c'est pas précisé si ce qui est retourné c'est le temps cpu totale ou moyen si c'est le temps cpu totale il faut le diviser par le nombre de thread sinon non (enfin c'est ce que je crois avoir compris)